### PR TITLE
update html head title to SDR

### LINKED ISSUE #23: Set the title value (in the html head)
Set it to `SDR | Stanford Digital Repository`. That's it. That's the issue. (You can disregard the comments below.)

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made?

Fixes #23 -- sets the html head title

## How was this change tested?

request spec

### CODE REVIEW

File: spec/requests/application_html_spec.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'ApplicationHtml', type: :request do
```
Comment (jcoyne) [inline spec/requests/application_html_spec.rb:6]: Would you mind changing this to be "the homepage" or similar and adjusting the file name?

Comment (jcoyne) [inline spec/requests/application_html_spec.rb:6]: Would you mind changing this to be "the homepage" or similar and adjusting the file name?

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: spec/requests/application_html_spec.rb (hunk 1 above)
Comment (ndushay) [reply spec/requests/application_html_spec.rb:6]: sure.  wasn't sure if this would apply all over the application or not.

**Comment by ndushay** [comment]
pushed amended commit, @jcoyne

File: Gemfile [hunk 2]
```diff
@@ -6,6 +6,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 group :development, :test do
   gem 'byebug'
   gem 'factory_bot_rails'
+  gem 'pry-byebug'
```
Comment (jcoyne) [inline Gemfile:9]: Can you extract this to a separate PR?  I really hate pry and I'd prefer if it was not part of the discussion on this issue.  Consider doing https://github.com/gerrywastaken/Gemfile.local maybe?

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 2 above)
Comment (ndushay) [reply Gemfile:9]: d'oh - that was a mistake.  sorry!

**Comment by ndushay** [comment]
@jcoyne ok - repaired my goofed commit.  this is now tightly focused on the task at hand

**Review by jcoyne — APPROVED** [review]
